pkg/cmd/admin/migrate/storage/storage.go

```diff
@@ -132,6 +132,10 @@ func (o *MigrateAPIStorageOptions) save(info *resource.Info, reporter migrate.Re
 	switch info.Object.(type) {
 	// TODO: add any custom mutations necessary
 	default:
+		// FIXME: This is happening for projectrequests/ resource
+		if len(info.Name) == 0 {
```

Comment:
I'm not sure why this suddenly started breaking - it worked before.  Add project requests to the exclude list and remove this code.